### PR TITLE
WIP: Added localization to dynamic package

### PR DESCRIPTION
**Pull request by dinodsaurus**

### CODE REVIEW

File: src/dynamicPackage/queries/AllDynamicPackages.js [hunk 1]
```diff
@@ -33,10 +33,11 @@ export default {
   resolve: async (
     ancestor: mixed,
     { simpleSearch, currency, ...pagination }: Object,
-    { dataLoader }: GraphqlContextType,
+    { dataLoader, locale }: GraphqlContextType,
   ) => {
     const packages = await dataLoader.dynamicPackages.load({
       currency,
+      locale,
```
Comment (michalsanger) [inline src/dynamicPackage/queries/AllDynamicPackages.js]: The dataloader does not need whole `locale`, use just the string `language` param plz

File: src/dynamicPackage/dataloaders/DynamicPackages.js [hunk 2]
```diff
@@ -22,6 +22,9 @@ export type SearchParameters = {|
   returnDate: Date,
   passengers: { adults: number, infants: number },
   currency: ?string,
+  locale: {
```
Comment (michalsanger) [inline src/dynamicPackage/dataloaders/DynamicPackages.js]: make it just `language: string`. The locale in context is always set

File: src/dynamicPackage/dataloaders/DynamicPackages.js [hunk 3]
```diff
@@ -62,6 +65,8 @@ export default class DynamicPackageDataLoader {
 
   async prepareParams(params: SearchParameters) {
     const apiKey = await this.getApiKey(params.fromAirport);
+    const lang = idx(params, _ => _.locale.language);
```
Comment (michalsanger) [inline src/dynamicPackage/dataloaders/DynamicPackages.js]: Once you make `language` param required you will not need this

**Comment by dinodsaurus** [comment]
@michalsanger I added a bit more robust parsing of the lang, and it's backward compatible with `frontend` repo

**Pull request merged**